Marlin/Configuration.h

```diff
@@ -49,7 +49,7 @@
  *
  * Note: Update also Version.h !
  */
-#define CONFIGURATION_H_VERSION 010100
+#define CONFIGURATION_H_VERSION 010101
```

Comment:
Could you point me in the right direction, I could not find information on this...